pkg/api/v1alpha1/cluster_types.go

```diff
@@ -70,6 +70,12 @@ func (c *Cluster) HasAWSIamConfig() bool {
 	return false
 }
 
+// IsPackagesEnabled checks if the configuration supports curated packages
+// installation.
+func (c *Cluster) IsPackagesEnabled() bool {
+	return !c.IsSelfManaged() && (c.Spec.Packages == nil || !c.Spec.Packages.Disable)
```

Comment:
Does this mean that packages can't be enabled in management clusters?

Reply:
No, they can be and are, it's just that it's handled by the CLI for management clusters.

Reply:
This is confusing to me then, why is there a method for `IsPackagesEnabled` then? That makes it sound like it should apply for any cluster that has packages enabled or not, regardless of if it's handled by the CLI.

Is `IsPackagesSupported` a better name based on the comment? I think understanding the use case better will help explain it.